langchain-core/src/callbacks/base.ts

```diff
@@ -259,6 +260,19 @@ abstract class BaseCallbackHandlerMethodsClass {
     tags?: string[]
   ): // eslint-disable-next-line @typescript-eslint/no-explicit-any
   Promise<any> | any;
+
+  handleCustomEvent?(
+    eventName: string,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    payload: any,
+    runId: string,
+    parentRunId?: string,
```

Comment:
Drop parentRunId b/c user can already define it, but we actually don't provide the correct one